kmip/demos/pie/add_attribute.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2015 The Johns Hopkins University/Applied Physics Laboratory
```

Comment:
Also, since you don't work for JHU/APL you don't need to include the JHU/APL copyright. Just include the Apache 2.0 license header (lines 4-14).